src/coreclr/jit/emitxarch.cpp

```diff
@@ -176,7 +176,7 @@ bool emitter::DoesWriteZeroFlag(instruction ins)
 //
 bool emitter::DoesResetOverflowAndCarryFlags(instruction ins)
 {
-    return (CodeGenInterface::instInfo[ins] & INS_FLAGS_Resets_CF_OF_Flags) != 0;
+    return (CodeGenInterface::instInfo[ins] & (Resets_OF | Resets_CF)) != 0;
```

Comment:
Doesn't this need to be:
```
return (CodeGenInterface::instInfo[ins] & (Resets_OF | Resets_CF)) == (Resets_OF | Resets_CF);
```
? I.e., the function says if BOTH are reset, but you're checking if only one is reset.

Reply:
Yes, that makes sense.